Formula/vim-gtk.rb

```diff
@@ -0,0 +1,72 @@
+class VimGtk < Formula
+  desc "Vi 'workalike' with many additional features"
+  homepage "https://www.vim.org/"
+  # vim should only be updated every 25 releases on multiples of 25
+  url "https://github.com/vim/vim/archive/v8.2.2650.tar.gz"
+  sha256 "0284b0e66287641b196b29496ea7d44ad5a728da0fc3b8eea36f1ed3636b661e"
+  license "Vim"
+  head "https://github.com/vim/vim.git"
+
+  depends_on "gtk+3"
+  depends_on "gettext"
+  depends_on "lua"
+  depends_on "perl"
+  depends_on "python@3.9"
+  depends_on "ruby"
+
+  uses_from_macos "ncurses"
+
+  conflicts_with "ex-vi",
+    because: "vim and ex-vi both install bin/ex and bin/view"
+
+  conflicts_with "macvim",
+    because: "vim and macvim both install vi* binaries"
```

Comment:
We also need a similar `conflicts_with "vim",` in this formula, and you need to add `vim-gtk` to `audit_exceptions/throttled_formulae.json`.

Please squash your changes together into one commit with the commit message
```
vim-gtk 8.2.2650 (new formula)
```
Thanks!

Reply:
> We also need a similar conflicts_with "vim"

What should i put as the reason?

> and you need to add vim-gtk to audit_exceptions/throttled_formulae.json

What number should i put?

I'm not very conversant with the formula syntax.

Reply:
> > We also need a similar conflicts_with "vim"
> 
> What should i put as the reason?

The same as for `macvim`.

> > and you need to add vim-gtk to audit_exceptions/throttled_formulae.json
> 
> What number should i put? What do they mean? and what are throttled formulae?

Just do the same thing as `vim` does, because this is essentially a copy of `vim`. Throttled formulae are formulae where we don't bump the version every time upstream cuts a new release.